addons/trenches/functions/fnc_getSurfaceTexturePath.sqf

```diff
@@ -54,13 +54,13 @@ private _result = if !(isNil QGVAR(automaticFileSetup)) then {
    if ((_surfaceType find "#Gdt" == -1) || {worldName == "Tanoa"}) then {
        _basePath = getText (configFile >> "CfgWorlds" >> worldName >> "surfaceTextureBasePath")
    };
-   
+
    [_surfaceType, _basePath, getText(configFile >> "CfgWorlds" >> worldName >> "filePrefix")] call _getTexturePath;
 };
 
-if (isNil _result || _result isEqualTo []) then {
-    _result = DEFAULT_TEXTURE;
+if !(_result isEqualType "") then {
```

Comment:
I don't see how that condition could ever be true.
_result comes from `_getTexturePath ` and that has no path that does not return a string.